deploy/charts/trust-manager/values.yaml

```diff
@@ -50,6 +50,8 @@ defaultPackageImage:
 app:
   # -- Verbosity of trust-manager logging; takes a value from 1-5, with higher being more verbose
   logLevel: 1
+  # -- Password for JKS and PKCS12 bundles pretection
+  pkcs12Password: defaultPKCS12Passwd
```

Comment:
Hello,
Thank you for you feedback, this PR is gonna be the 2nd one for this project!
Yes, I agree with you, I change it, however related tests were added, so there are two tests now: password-less and arbitrary password for both Secret and ConfiMap resources.

Reply:
>  related tests were added, so there are two tests now: password-less and arbitrary password for both Secret and ConfiMap resources.

Ahhh, my review comment was not good - it could easily be misinterpreted. What I meant, is that to get the Helm Deployment template correct, you probably have to try a few rounds of local rendering with different Helm values.